.github/workflows/integration_test.yml

```diff
@@ -35,14 +35,16 @@ jobs:
           kubectl logs $(kubectl -n kepler get pods -o name) -n kepler
           kubectl get all -n kepler
       
-      - name: run integration_test
+      # TODO: need to separate bcc built if libbpf base is created and applied
+      - name: run integration_test for libbpf built
         run: |
           docker ps -a
           mkdir -p /tmp/.kube
           kind get kubeconfig --name=kind > /tmp/.kube/config
           while true; do kubectl port-forward --address localhost -n kepler service/kepler-exporter 9102:9102; done &
           kubectl logs -n kepler daemonset/kepler-exporter
           kubectl get pods -n kepler -o yaml
-          go test ./e2e/... --tags bcc -v --race --bench=. -cover --count=1 --vet=all
+          make e2e-bcc
```

Comment:
I will separate them into two tests later when the libbpf base image is ready.  

In this draft, the base image for libbpf is not available so I just add dependencies of libbpf to the bcc base image. When calling make e2e-bcc, it will tag only bcc and libbpf library will not be loaded, and vice versa for e2e-libbpf.

Reply:
updated by separating integration test for libbpf to another file (integration_test_libbpf.yml).